packages/scripts/src/generate-master-list/generator.ts

```diff
@@ -243,6 +350,12 @@ export const generateMasterList = async (verbose = false): Promise<void> => {
       { path: pathToTargets + 'README.md', text: target.text },
       { path: 'MASTERLIST.md', text: allAdapterText },
     ])
+
+    try {
+      await buildAirtable(allAdaptersTable)
+    } catch (e) {
+      console.warn(`Error updating AirTable. `, e)
+    }
```

Comment:
Another concern I have is that this couples the airtable updates with the usage of `yarn generate:master-list`, even if it's only being generated to test. IMO we should at least pass a flag to specify whether to generate the airtable, and use that flag in `.github/workflows/consume-changesets.yml`